lib/content-services/src/lib/category/services/category-tree-datasource.service.spec.ts

```diff
@@ -96,15 +96,15 @@ describe('CategoryTreeDatasourceService', () => {
                         parentId: 'parent id 1',
                         level: 0,
                         nodeType: TreeNodeType.RegularNode,
-                        hasChildren: false,
+                        hasChildren: true,
                         isLoading: false
                     }, {
                         id: 'some id 2',
                         nodeName: 'Language/some other name',
                         parentId: 'parent id 2',
                         level: 0,
                         nodeType: TreeNodeType.RegularNode,
-                        hasChildren: false,
+                        hasChildren: true,
```

Comment:
can we test true and false case for that?
hasChildren: res.entry.hasChildren

Reply:
added